app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_of_file = command.split()
+    if name_of_file[0] != "cp" or name_of_file[1] == name_of_file[2]:
+        return
+
+    from shutil import copyfile
```

Comment:
Is your idea to do import only `if` statement is False?:) Cuz I don't get in what other cases could this import be here.
It certainly makes your code more unstructured. But  do you have any benefits?

Reply:
It was intended that way, but in reality there will be fewer cases where the function will stop on `if statement`, so bad  structure does not override by single absences of import